sigstore-java/src/main/java/dev/sigstore/KeylessSigner.java

```diff
@@ -37,34 +40,49 @@
 import java.security.SignatureException;
 import java.security.cert.CertificateException;
 import java.security.spec.InvalidKeySpecException;
+import java.time.Duration;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Map;
+import org.checkerframework.checker.nullness.qual.Nullable;
```

Comment:
javax.annotation is the one we've been using everywhere ?

Reply:
`javax.annotation` is not `type` annotation. Currently, sigstore-java uses both checkerframework and jsr305.

The issue with jsr305 is the ideal annotation there is `@CheckNull` which is not very intuitive.
The meaning is that in jsr305, `@Nullable` means "might be null" while `@CheckNull` means "definitely can be null, so everybody should check for nulls".

Reply:
ah right, kk thanks for clarifying.